test/activation_based/test_inductor_cache.py

```
@@ -116,11 +114,9 @@ def fake_compile(fn, **_kwargs):
     node(torch.randn(4, 2, 3))
     assert inductor_cache.info()["entries"] == 1
 
-    copied = copy.deepcopy(node)
-    restored = pickle.loads(pickle.dumps(node))
+    copy.deepcopy(node)
+    pickle.loads(pickle.dumps(node))
```

Ruff (0.16.1)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)